python/semantic_kernel/connectors/memory/qdrant/qdrant_utils.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import datetime
+import uuid
+import struct
+
+import qdrant_client
+from qdrant_client.models import Filter, FieldCondition, MatchValue
+
+from numpy import array, linalg, ndarray, zeros
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+from typing import Optional
+
+"""
+Utility function(s) for Qdrant vector database to support Qdrant Semantic Kernel memory implementation.
+"""
+
+
+def guid_comb_generator() -> str:
+    """
+    Generate a GUID-comb identifier.
+
+    Returns:
+        str: A GUID-comb identifier.
+    """
+
+    guid_bytearr = bytearray(str(uuid.uuid4()), "utf-8")
```

Comment:
Is there a specific reason you're expanding on uuid.uuid4? uuid4 should be sufficient.

Reply:
Hi this was updated with commit: https://github.com/microsoft/semantic-kernel/pull/919/commits/5ab6943282497ed1464cc0626824b616670d4b22 for comb